biz.ganttproject.app.libs/build.gradle

```diff
@@ -2,6 +2,7 @@ buildscript {
     ext.kotlin_version = '1.3.+'
     repositories {
         mavenCentral()
+        maven { url 'https://dl.bintray.com/kotlin/kotlin-eap'}
```

Comment:
This issue is already fixed in master. You need to git merge master into this branch.